src/components/Form/FormFields/PhoneNumberFormField.tsx

```diff
@@ -17,157 +16,146 @@ import {
 } from "../FieldValidators";
 import CareIcon from "../../../CAREUI/icons/CareIcon";
 import { Popover, PopoverButton, PopoverPanel } from "@headlessui/react";
-import { useTranslation } from "react-i18next";
-
+import React from "react";
 const phoneCodes: Record<string, CountryData> = phoneCodesJson;
 
 interface Props extends FormFieldBaseProps<string> {
+  myfunc?: (er: FieldError) => void;
```

Comment:
what is `myfunc`? can we use proper names for the props so that it's understandable?

Reply:
@rithviknishad it will show error when any error appear in phone number field or any serchfield in the serchbyfieldComponent

Reply:
https://github.com/ohcnetwork/care_fe/pull/8834#issuecomment-2439930727

Reply:
you don't need to modify PhoneNumberFormField for that. Just check the value in the search by field component and show error from there.

Reply:
@rithviknishad  What is the problem?I tried to do this but 
To access value from phcompo to serchcompo using the ref  is creating problem and complexity

Reply:
@rithviknishad ,I have define a type of instead of this https://github.com/ohcnetwork/care_fe/pull/8834#discussion_r1822685003
, please check

Reply:
This is still unresolved...